src/root/components/highlighted-operations/operation-card.js

```diff
@@ -67,9 +67,9 @@ const OperationCard = ({operationId, operationName, emergencyDeployments, appeal
 
         <div className='card_box_container card_box_container--op'>
           <div className='card_box card_box_left card_box--op'>
-            <div className="card_box_no">{requested && requested !== 0 ? `${n(requested)} CHF` : '--'}</div>
+            <div className="card_box_no">{requested && requested !== 0 ? `${n(requested)}` : '--'}</div>
```

Comment:
@imohkay - is the removing of `CHF` here intentional?

Reply:
@batpad So the only reason I did it was because the new font sizes were moving text to the next line, and in the Zeplin designs, I realised that the CHF is on the next line anyway. I can easily revert if not desired.